docusaurus/docusaurus.config.js

```diff
@@ -131,6 +133,9 @@ module.exports = {
       apiKey: 'MOCK',
       indexName: 'MOCK',
     },
+    colorMode: {
```

Comment:
what is this doing?

Reply:
Removes the dark/light switch mode since we don't have a dark palette at the moment, Josh asked me to deactivate it for now

Reply:
oh ok 😢

Reply:
RIP 😞